packages/joint-core/src/dia/groups/CellLayer.mjs

```diff
@@ -0,0 +1,37 @@
+import { CellGroupCollection, CellGroup } from './CellGroup.mjs';
+
+export class CellLayerCollection extends CellGroupCollection {
+
+    // `comparator` makes it easy to sort cells based on their `z` index.
+    comparator(model) {
+        return model.get('z') || 0;
+    }
+}
+
+export class CellLayer extends CellGroup {
+
+    defaults() {
+        return {
+            type: 'CellLayer',
+            collectionConstructor: CellLayerCollection,
```

Comment:
Do we need to store a JS class inside the model attributes (`CellGroup` is a model)?